static/css/base.css

```diff
@@ -0,0 +1,649 @@
+body {
+  top: 0;
+  left: 0;
+  color: rgba(51,51,51,1);
+}
+
+h1 {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.75rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h2 {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.75rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h3 {
+  font-family: "Roboto Slab", serif;
+  font-size: 1.5rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h4 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 1.25rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+h5 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 1rem;
+  line-height: 125%;
+  font-weight: normal;
+}
+
+h6 {
+  font-family: "Open Sans", sans-serif;
+  font-size: 0.875rem;
+  line-height: 125%;
+  font-weight: normal;
+  color: rgba(51,51,51,1);
+}
+
+.header-text {
+  font-weight: normal;
+  color: rgba(250,250,250,1);
+  text-align: center;
+
+}
+
+.p-latestblog {
+  font-size: .875rem;
+}
+
+.font-weight-bolder {
+  font-weight: 600;
+}
+
+.offwhite-text {
+  color: rgba(250,250,250,1); /* #FAFAFA */
+}
+
+.blue-text {
+  color: rgba(0,97,142,1); /* #00618E */
+}
+blue {
```

Comment:
Can someone confirm that this selector exists because we're actually using a custom TAG named `blue` rather than a typo for a CLASS named `blue`?

Reply:
@cameroneshgh can you confirm this?

Reply:
This is intentional (from the home page in the trending assets):
![image](https://user-images.githubusercontent.com/11238397/42225564-d2d05d8c-7eaa-11e8-9bc6-e4a8ba650dbe.png)